lib/scopes/form526_submission_state.rb

```diff
@@ -112,7 +112,7 @@ module Form526SubmissionState
 
       scope :failure_type, lambda {
         # filtering in stages avoids timeouts. see doc for more info
-        allids = all.pluck(:id)
+        allids = all.pluck(:id).where(submitted_claim_id: nil)
```

Comment:
small optimization. we still want it at the end to address [the edge case documented here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/engineering_research/526_scopes.md#failure_type-edge-case)